bot/resources/fun/madlibs_templates.json

```diff
@@ -0,0 +1,135 @@
+                      " it over.\""]
+        },
+        {
+            "title": "The Fun Park",
+            "blanks": ["adjective", "plural noun", "noun", "adverb", "number", "past tense verb", "adjective ending in -est", "past tense verb", "adverb", "adjective"],
+            "value": ["Today, my fabulous camp group went to a(an) ", " amusement park. It was a fun park with lots of cool ",
+                      " and enjoyable play structures. When we got there, my kind counselor shouted loudly, \"Everybody off the ",
+                      ".\" My counselor handed out yellow tickets, and we scurried in. I was so excited! I couldn't figure out what exciting thing to do first. I saw a scary roller coaster I really liked so, I ",
+                      "  ran over to get in the long line that had about ", " people in it. when I finally got on the roller coaster I was ",
+                      ". In fact, I was so nervous my two knees were knocking together. This was the ", " ride I had ever been on! In about two minutes I heard the crank and grinding of the gears. Thats when the ride began! When I got to the bottom, I was a little ",
+                      " but I was proud of myself. The rest of the day went ", ". It was a ", " day at the fun park."]
+        },
+        {
+            "title": "A Spooky Campfire Story",
+            "blanks": ["adjective", "adjective", "number", "adjective", "animal", "noun", "animal", "name", "verb", "adjective", "adjective"],
+            "value": ["Every summer, I get totally amped and ", " to go camping in the deep, ", " forests. It's good to get away from it all - but not too far, like getting lost! Last year, my friend and I went hiking and got lost for ",
+                      " hour(s). We started off on a(n) ", " adventure, but we kept losing the trail. Night began to fall, and when we heard the howls of a ",
+                      ", we began to panic. It was getting darker and our flashlights were running on ", ". I'm sure glad my pet ", ", ", ", was with us. He is one gifted creature, because he was able to guide us back by ",
+                      " the ", " s'mores by the campfire. This year, before setting off on an ", " journey, I'll be sure to have working flashlights - and of course, my gifted pet!"]
+        },
+        {
+            "title": "Weird News",
+            "blanks": ["noun", "place", "verb ending in ing", "noun", "name", "verb", "noun", "verb", "noun", "part of body", "type of liquid", "place", " past tense verb ", "foreign country", "verb", "noun", "past tense verb", "adjective", "verb", "noun", "plural noun"],
+            "value": ["A ", " in ", " was arrested this morning after he was caught ", " in front of ", ". ", " had a history of ", ", but no one-not even his ", "-ever imagined he'd ", " with a ", " stuck in his ", ". After drinking a ", ", cops followed him to a ",
```

Comment:
```suggestion
            "value": ["A ", " in ", " was arrested this morning after he was caught ", " in front of ", ". ", " had a history of ", ", but no one-not even his ", "ever imagined he'd ", " with a ", " stuck in his ", ". After drinking a ", ", cops followed him to a ",
```

Reply:
The dash is supposed to be there because:

"but no one - not even his _____ - ever imagined..."

Does that make sense? @Bluenix2

Reply:
Ah, well look how you added spaces around the dash.

Reply:
Yes, I did notice. Will fix.